keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -1114,6 +1115,11 @@ We plan to add e2e test under api-machinery for crd expression validation:
 - Optimization on super-linear complexity growth
 - Adding metric of the latency of CEL evaluation for CRD evaluation
 
+#### GA
+
+- Formatted message
```

Comment:
Add note here that this is targeted for 1.27 and that we must not graduate until 1 release after this feature is added?